azure-mgmt-keyvault/src/main/java/com/microsoft/azure/management/keyvault/Vault.java

```diff
@@ -108,6 +108,14 @@
      */
     @Beta(SinceVersion.V1_11_0)
     CreateMode createMode();
+    
+    /**
+     * Get the networkAcls value.
+     * 
+     * @return the networkAcls value
+     */
+    @Beta(SinceVersion.V1_11_0)
+    NetworkRuleSet networkAcls();
```

Comment:
@tiffanyachen In Fluent we try to avoid using acronyms especially when these are not needed. In this the method can be replaced with "networkRuleSet()". It's also consistent with a similar class found in Storage management library.